game/src/Actors/Player.gd

```diff
@@ -20,6 +20,7 @@ func _physics_process(delta: float) -> void:
 	_velocity = move_and_slide_with_snap(
 		_velocity, snap, FLOOR_NORMAL, true
 	)
+	die() if $".".position.y > 1080.0 else ""
```

Comment:
The like reads as `die if when getting myself as a node path, Y position is > 1080 else empty string`

It'd be way clearer as something like:

```gdscript
if position.y > level_limit:
    die()
```

In any case, I'd rather use an area2D with the line collider. It creates an infinite horizontal limit that you can move up and down in the level.